client-src/elements/chromedash-gate-column.js

```diff
@@ -462,12 +493,24 @@ export class ChromedashGateColumn extends LitElement {
   }
 
   renderQuestionnaireSkeleton() {
-    return nothing;
+    return html`
+      <h2>Survey questions</h2>
+      <div id="questionnaire">Loading...</div>
+      <p class="instructions">&nbsp;</p>
+    `;
   }
 
   renderQuestionnaire() {
-    // TODO(jrobbins): Implement questionnaires later.
-    return nothing;
+    const questionnaireText = GATE_QUESTIONNAIRES[this.gate.gate_type];
+    const instructions = (
+      questionnaireText ?
+        html`Please post responses in the comments below.` :
```

Comment:
```suggestion
        Please post responses in the comments below. :
```
Does it need html tags?

Reply:
I think that the `&nbsp;` needs it, so I used it for the other one to make the two conditional branches consistent.